sefaria/model/__init__.py

```diff
@@ -10,6 +10,8 @@
 """
 
 import abstract
+import django
```

Comment:
Did you intend to put this here?  
I don't think that we want this at the model layer.